### PR TITLE
docs: Add KVS keyboard property

### PR DESCRIPTION
**Pull request by mcdurdin**
Documentation for keymanapp/keyman#5749. I rewrote the whole page as markdown and added a bunch of missing properties. The page is still too lightweight but is a step better than before...

### CODE REVIEW

File: developer/engine/web/15.0/reference/keyboard_properties.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+: `string`, Keyboard help, in HTML; if present replaces `KV` in the On Screen Keyboard, optional
+
+* `KM`
+: `number`, `1` to use mnemonic layout, `0` or omitted for positional, optional
+
+* `KBVER`
+: `string`, version of the keyboard (should be dotted decimal format), optional
+
+* `KMBM`
+: `number`, bitmask denoting modifiers used in the keyboard
```
Comment (darcywong00) [inline developer/engine/web/15.0/reference/keyboard_properties.md:40]: Is this required or optional?

File: developer/engine/web/15.0/reference/keyboard_properties.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+: `object`, touch layout definition, optional.
+
+* `KVER`
+: `string`, version of Keyman Developer used to compile the keyboard, optional
+
+* `KVS`
+: `string[]`, array of all variable store names found in the keyboard (15.0 and later), optional
+
+* `KS`
+: `number`, `1` means Unicode characters U+10000-U+10FFFF are used in the keyboard, optional
```
Comment (darcywong00) [inline developer/engine/web/15.0/reference/keyboard_properties.md:52]: Can we keep "SMP" somewhere in this description?

Comment (mcdurdin) [reply developer/engine/web/15.0/reference/keyboard_properties.md:52]: Technically SMP refers only to plane 1... but can add it back.

Comment (mcdurdin) [reply developer/engine/web/15.0/reference/keyboard_properties.md:52]: image
From wikipedia...

File: developer/engine/web/15.0/reference/keyboard_properties.md (hunk 1 above)
Comment (mcdurdin) [reply developer/engine/web/15.0/reference/keyboard_properties.md:40]: optional

**Review by darcywong00 — APPROVED** [review]
lgtm

**Pull request merged**